### PR TITLE
enable overide retain flag in bridge

### PR DESCRIPTION
**Pull request by OdyWayne**

### CODE REVIEW

**Review by RanMaoyi — APPROVED** [review]

**Review by wanghaEMQ — APPROVED** [review]

File: etc/nanomq_example.conf [hunk 1]
```diff
@@ -702,6 +702,12 @@ bridges.mqtt.emqx1 {
 			# # This must present to enable the forwarding of bridging 
 			# # Value: String
 			local_topic = "topic1"
+			# # Retain is used to override the ratain flag in the msg is about
+			# # to forward to remote. Set 0 or 1 to override the ratain flag. 
+			# # Leave it as it is by not set it.
+			# # Value: 0 | 1
+			# # Default: not set
+			# # retain = 0
```
Comment (wanghaEMQ) [inline etc/nanomq_example.conf:710]: Why not tell user retain=2 means not override

**Review by lee-emqx — APPROVED** [review]

File: etc/nanomq_example.conf (hunk 1 above)
Comment (JaylinYu) [reply etc/nanomq_example.conf:710]: nice try

**Review by JaylinYu — APPROVED** [review]
remember to give old conf same feature

**Comment by OdyWayne** [comment]
> remember to give old conf same feature

ok

**Pull request merged**